File: PlayerClass/player.py
class Player:
    name = "Defaultskin"
    level = 1
    maxxp = (level * 3) ** 2
    maxhp = 1
    hp = maxhp
    st = 1
    mp = 0
    ag = 1
    xp = 0
    score = 0
    agXp = 0
    agMax = (ag * 3) ** 1.1

    def heal(self):
        self.hp = self.maxhp

    def setXp(self):
        self.maxxp = (self.level * 3) ** 2
    def setAgXp(self):
        self.agMax = (self.ag * 3) ** 1.1


class Mage(Player):
    klass = "Mage"
    maxhp = 10
    hp = maxhp
    st = 1
    mp = 5
    pass


class Rouge(Player): #a really worthless "class" atm
    klass = "Rouge"
    maxhp = 10
    hp = maxhp
    st = 3
    mp = 3
    pass


class Warrior(Player):
    klass = "Warrior"
    maxhp = 15
    hp = maxhp
    st = 5
    pass

class Bugfix(Player):
    klass = "Haxors"
    maxhp = 1500
    hp = maxhp
    st = 500
    pass

File: src/playerClass/Player.java
package playerClass;

public class Player extends OVclass{

    public Player(int klass){
        super();
        if(klass == 1){
            this.name = "Mage";
            maxHp = 10;
            hp = maxHp;
            st = 1;
            mp = 5;
        }else if(klass == 2){
            this.name = "Warrior";
            maxHp = 15;
            hp = maxHp;
            st = 5;
        }else if(klass == 3){
            this.name = "Rouge";
            maxHp = 10;
            hp = maxHp;
            st = 3;
            mp = 3;
        }else{
            System.out.println("error in klass chooser");
        }
    }

    public void PrintStats(){
        System.out.println("Class: " + this.name);
        System.out.println("Level: " + this.level);
        System.out.println("HP: " + this.hp +"/"+ this.maxHp);
        System.out.println("ST: " + this.st);
        System.out.println("MP: " + this.mp);
        System.out.println("AG: " + this.ag);
    }
}
